torch/utils/data/datapipes/iter/combinatorics.py

```diff
@@ -104,8 +103,9 @@ def buffer_replace(buffer, x):
         buffer[idx] = x
         return val
 
-    def set_shuffle_settings(self, shuffle=True):
+    def set_shuffle(self, shuffle=True):
```

Comment:
Another suggestion I have for this would be `enable_shuffle()` or `enable_shuffling()`. But no strong opinion.